pkgs/development/python-modules/reprint/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  buildPythonPackage,
+  colorama,
+  fetchFromGitHub,
+  lib,
+  setuptools,
+  six,
+  stdenv,
+}:
+
+buildPythonPackage rec {
+  pname = "reprint";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "Yinzo";
+    repo = "reprint";
+    rev = "${version}";
+    hash = "sha256-99FC12LcvvRRwNAxDSvWo9vRYmieL0JHSaCJqO/UGEs=";
+  };
+  doCheck = false; # Disable tests due to unnecessary dependency on backports.shutil_get_terminal_size
```

Comment:
Why not drop the dep then? we want to run tests.

Also doCheck should be between dependencies and pythonImportsCheck

Reply:
There are actually no tests so I will skip the test phase